app/frontend/wallet/cardano-trezor-crypto-provider.js

```diff
@@ -213,24 +213,22 @@ const CardanoTrezorCryptoProvider = (CARDANOLITE_CONFIG, walletState) => {
     return data
   }
 
-  async function signTx(unsignedTx) {
+  async function signTx(unsignedTx, rawInputTxs) {
     const inputs = []
     for (const input of unsignedTx.inputs) {
       inputs.push(await prepareInput(input))
     }
 
-    const transactions = []
-    for (const input of inputs) {
-      const transaction = (await blockchainExplorer.fetchTxRaw(input.prev_hash)).toString('hex')
-      transactions.push(transaction)
-    }
-
     const outputs = []
     for (const output of unsignedTx.outputs) {
       const data = await prepareOutput(output)
       outputs.push(data)
     }
 
+    transactions = []
```

Comment:
You forgot const. I would also refactor this to:
`
const rawInputTxsAsStrings = rawInputTxs.map((tx) => tx.toString('hex'))
`